probe/kubernetes/deployment.go

```diff
@@ -0,0 +1,71 @@
+package kubernetes
+
+import (
+	"fmt"
+
+	"github.com/weaveworks/scope/report"
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/unversioned"
+	"k8s.io/kubernetes/pkg/apis/extensions"
+	"k8s.io/kubernetes/pkg/labels"
+)
+
+// These constants are keys used in node metadata
+const (
+	DeploymentID                  = "kubernetes_deployment_id"
```

Comment:
Given we don't merge nodes of different types anymore, is it worth have a meta.GetNode() and unifying all the metadata keys between deployments, pods, services, rs, rc etc etc?